test/e2e-secondary-network/secondary_network_test.go

```diff
@@ -0,0 +1,245 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"fmt"
+	"log"
+	"net"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	logs "github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v3"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+
+	antreae2e "antrea.io/antrea/test/e2e"
+)
+
+// Structure to extract and store the secondary network configuration information parsed from secondary_network_config_param.yml.
+type PodConfig struct {
+	InterfaceType struct {
+		InterfaceType string `yaml:"interfacetype"`
+	} `yaml:"interface_type"`
+	SriovConf struct {
+		NetworkInterface string `yaml:"networkinterface"`
+		NumberOfVfs      int    `yaml:"numberofvfs"`
+	} `yaml:"sriov_conf"`
+	VirNet struct {
+		TotalNumberOfVirtualNetworks int      `yaml:"totalnumberofvirtualnetworks"`
+		VirtualNetworknames          []string `yaml:"virtualnetworknames"`
+	} `yaml:"vir_net"`
+	CreatePod struct {
+		NumberOfPods int             `yaml:"numberofpods"`
+		Describe     [][]interface{} `yaml:"describe"`
+	} `yaml:"create_pod"`
+}
+
+var service PodConfig
+
+// Structure for extracting the variables for describing the Pod from secondary_network_config_param.yaml file.
```

Comment:
Do you mean secondary-network-configuration.yml? I did not find `secondary_network_config_param.yaml` in the repo.

Reply:
took care